sdks/python/tests/unit/message_processing/test_message_processors.py

```diff
@@ -0,0 +1,45 @@
+from unittest import mock
+
+from opik.message_processing import message_processors
+
+from . import common
+
+
+def test_process_create_trace_batch_message():
```

Comment:
If you are reading this test for the first time it might be quite hard to understand what's the expected behavior here.

Suggestions:
1. Make it obvious that one trace in the batch is approximately 1 MB (I'd update the create function name and/or signature for that)
2. Declare in the test name that we expect message processor to split the batch into minibatches and push them back.

Ideally for the non-trivial behavior tests, we should stick to the format of the names like:
test_TESTED_NAME__CONDITION__EXPECTED_RESULT.

Tests can be very verbose, it helps to understand the behavior even before looking into implementation.